airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -527,7 +527,7 @@ def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[Task
                         ti.queue,
                         ti.command_as_list(),
                         ti.executor_config,
-                        ti.prev_attempted_tries,
+                        ti.try_number,  # prev attempted tries
```

Comment:
```suggestion
                        ti.try_number,
```

Not sure we need this comment?

Reply:
yeah.....